src3/nmbguessapp.js

```diff
@@ -0,0 +1,61 @@
+let guessCounter = 3;
+let guessValue = 0;
```

Comment:
You need to use the compareNumber (you named it testFunction) in this file to compare numbers. Even though the logic you used makes the game work, you need to break some of that work into the function so that it isn't all in the eventListener. Since checking the relative value of a guess is a repeated action, its cleaner and more efficient for it be in a function rather than using the if/else statements.